eng/Packages.props

```diff
@@ -38,6 +38,8 @@
     <PackageReference Update="Microsoft.Azure.KeyVault" Version="3.0.4" />
     <PackageReference Update="Microsoft.Azure.Kusto.Ingest" Version="8.1.0" />
     <PackageReference Update="Microsoft.Azure.Management.Fluent" Version="1.31.0" />
+    <PackageReference Update="Microsoft.Azure.Management.ResourceManager.Fluent" Version="1.37.1" />
+    <PackageReference Update="Microsoft.Azure.Management.Storage" Version="19.0.0" />
```

Comment:
Why use all the crusty old API's that aren't supported anymore?

Reply:
The RM.Fluent one is basically for the ResourceId type, so I can parse the resource ids in a sane way. I haven't found that anywhere else. Is there a "new" version of the storage management libraries? I have only seen the libraries to talk to storage, not manage it.

Reply:
https://azure.github.io/azure-sdk/releases/latest/mgmt/dotnet.html lists [Azure.Resourcemanager.Storage](https://www.nuget.org/packages/Azure.ResourceManager.Storage/1.0.0-preview.2) as the latest one.

Reply:
that one is still -preview.2, I'd rather not use it yet if they haven't released it.